am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -31,23 +31,23 @@ void setupTest() {
     }
 
     @Test
-    void filterResource_whenRowExistWithAccessorIdAndResourceId_ReturnPassedJsonObject() {
-        ams.createResourceAccess(createRecord(resourceId, ACCESSOR_ID, EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS));
+    void whenRowExistWithAccessorIdAndResourceIdReturnPassedJsonObject() throws TransactionRolledbackException {
+        Map<JsonPointer, Set<Permission>> singleAttributePermission = new ConcurrentHashMap<>();
+        singleAttributePermission.put(JsonPointer.valueOf("/"), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS);
```

Comment:
Let's stop using "/" as test JSON path since it is almost impossible to have in real live. I am happy to have "" or "/name" for testing purposes. IMO the more real simulated scenario is the easier it is for reader to grasp what test is for.